src/tutorial/errors.md

```diff
@@ -120,15 +118,15 @@ It's the default return value of the function and means
 Why is this not written as `return Ok(());`?
 It easily could be – this is totally valid as well.
 The last expression of any block in Rust is its return value,
-and it is customary to omit needless `return`s.
+and it is customary to omit a needless `return`.
 
 </aside>
 
 ## Question Mark
 
 Just like calling `.unwrap()` is a shortcut
 for the `match` with `panic!` in the error arm,
-we have another shortcut for the `match` that `return`s in the error arm:
+we have another shortcut for the `match` that returns in the error arm:
```

Comment:
I *think* I prefer the old version of this.

Reply:
I can change it if preferred. The issue I had is that `return`s looked very weird rendered because the code formatting was smaller and the s looked like a capital S. I also felt like "returns" is a well-understood concept in coding, so it seemed unnecessary to make it code, not to mention that we use "returns" in a similar fashion elsewhere. If you're not persuaded, I'm happy to flip it back ;)

Reply:
My thought is more around the principle.  Are we ok with adding `s`, `ed`, and other suffixes to literals?  If so, then we shouldn't bother changing.  For the most part, I write in the style of doing that.

Reply:
When in doubt, leave it alone! I switched it back.

Reply:
Oops, messed up my branch. Will fix...